web/client/components/data/featuregrid/toolbars/Toolbar.jsx

```diff
@@ -115,4 +115,11 @@ module.exports = ({events = {}, mode = "VIEW", selectedCount, hasChanges, hasGeo
             visible={mode === "VIEW"}
             onClick={events.sync}
             glyph="map-filter"/>
+        <TButton
+            id="grid-map-chart"
+            tooltip={<Message msgId="featuregrid.toolbar.createNewChart"/>}
+            disabled={disableToolbar}
+            visible={mode === "VIEW" && widgetAvailable && !chartDisabled}
```

Comment:
`showChartButton`  may be a unique variable enhancer that is created in the selector as widgetAvailable && !chartDisabled

Reply:
right, i'll update it